### ISSUE TITLE
Sharing config for many nodes

### ISSUE THREAD

**Issue by rnavarette** [groupon/Selenium-Grid-Extras#84]
I'm wondering if there is an easy way to configure a node to use a specified shared config folder from the hub?  We're looking into managing and auto scaling a large grid and it would be great to be able to specify that an auto created node would use say "config1" or "config2", rather than relying on the machine name folder exist on the hub.

Thanks...

**Comment by smccarthy** [comment]
@rnavarette  So config1 might be for Chrome, config2 might be for Firefox, config3 might be for IE9, config4 might be for IE10, etc ?

So in this case you only had 4 configs , instead of the number of nodes.

I just wanted to make sure I understood your request clearly.

**Comment by rnavarette** [comment]
Yep, thats the idea.  Then, I might be able to spin up say 5 nodes of each.

**Comment by smccarthy** [comment]
I like the idea.

**Comment by rnavarette** [comment]
I've got more :)

Seriously though, is this the place for these discussions or is there a forum or mailing list?  I think this project makes the node configuration WAY easier than SG out of the box.  However, I think there is quite a bit more than could be done yet to make it even better.

I'd love to commit where I can as we put together our solution, but a good first step would be having a solid group vet some of the ideas and see where they best fit.

**Comment by smccarthy** [comment]
There is no mailing list or forum specific to Selenium-Grid-Extras. Though you can usually find me in #selenium on irc.freenode.net .
